ucm2/conf.d/sc8280xp/sc8280xp.conf

```diff
@@ -0,0 +1 @@
+../../Qualcomm/sc8280xp/SC8280XP-LENOVO-X13S.conf
```

Comment:
only one link from conf.d to ../../Qualcomm/sc8280xp/sc8280xp.conf .

Reply:
sure,I will make only one link  conf.d/sc8280xp/sc8280xp.conf to ../../Qualcomm/sc8280xp/LENOVO-X13S.conf